source/arch/begin_run_event.json

```diff
@@ -0,0 +1,50 @@
+{
+    "properties": {
```

Comment:
I think you are misreading the schema. "Properties" is just a list of the fields; it is not something we actually implement.